pkg/virtualization/core/storage/disk/logicaldisk.go

```diff
@@ -35,3 +36,14 @@ func (ld *LogicalDisk) GetDrive() (*resourceallocation.ResourceAllocationSetting
 	}
 	return resourceallocation.NewResourceAllocationSettingData(inst)
 }
+
+func (ld *LogicalDisk) GetPath() (string, error) {
+	value, err := ld.GetProperty("HostResource")
+	if err != nil {
+		return "", err
+	}
+	for _, hr := range value.([]interface{}) {
+		return hr.(string), nil
+	}
+	return "", errors.Wrapf(errors.NotFound, "")
```

Comment:
Can you add a descriptive error. Just not found is going to confuse to users.

Reply:
Also please add unit tests

Reply:
I ended up not requiring this method since we are getting the `Path` from the DVD drive directly. Let me remove this function.